test/README.md

```diff
@@ -46,11 +46,13 @@ Very low coverage.
 To execute the test suite run
 
     cd test/php
-    phpunit ../
+    UNIT_TEST_DSN='pgsql:dbname=nominatim_unit_tests' phpunit ../
```

Comment:
I have a mild preference to have the tests available by default and allow to switch them off via a parameter. Or would that be too dangerous because of the deletion?

Reply:
It's the default now. But added a check that it contains `unit_test` which makes it more unlikely somebody points it at an database.